app/views/groups/_group.html.erb

```diff
@@ -0,0 +1,11 @@
+<div id="<%= dom_id group %>" class="groupContainer">
+  <img src=<%= group.Icon %> alt="icon">
+  <div class="linkBox">
+    <%= link_to group.Name, group_expenses_path(group.id) %>
+  </div>
+  <% @total = 0 %>
+  <% group.groupsexpenses.each do |exp| %>
+    <% @total += exp.expenses.Amount %>
+  <% end %>
+  <div class="total"> <%= @total %> $ </div>
```

Comment:
- [ ] Please use the correct font, color and also make the expense amount bold as it is in the design guideline

### Design Guideline
![image](https://user-images.githubusercontent.com/46839548/219160649-839b9c37-49a1-4d17-b225-7845f60a167f.png)
### Your Design
![image](https://user-images.githubusercontent.com/46839548/219163969-307377be-c515-4625-9048-dafb6d7032ce.png)